index.html

```diff
@@ -159,141 +156,401 @@ <h2>
+                  </ol>
+                </li>
+                <li>If [=options=] are not equivalent to the current
+                [=pointer-lock options=] and [=options=] are supported then
+                perform the following steps
+                  <ol>
+                    <li>Update the user agent's [=pointer-lock target=] to this
+                    element.
+                    </li>
+                    <li>Update the user agent's [=pointer-lock options=] to
+                    [=options=]
+                    </li>
+                    <li>[=Queue an element task=] on the [=user interaction
+                    task source=], given this, to perform the following steps:
+                      <ol type='i'>
+                        <li>[=Fire an event=] named {{pointerlockchange}} at
+                        [=this=]'s [=Node/node document=].
+                        </li>
+                        <li>The <var>promise</var> must be resolved.
+                        </li>
+                      </ol>
+                    </li>
+                    <li>Return <var>promise</var>.
+                    </li>
+                  </ol>
+                </li>
+              </ol>
+            </li>
+            <li>If the request was not started from an [=engagement gesture=]
+            and the {{Document}} has not previously released a successful Pointer
+            Lock with {{Document/exitPointerLock()}}:
+              <ol>
+                <li>[=Queue an element task=] on the [=user interaction task
+                source=], given this, to perform the following steps:
+                  <ol type='i'>
+                    <li>[=Fire an event=] named {{pointerlockerror}} at
+                    [=this=]'s [=Node/node document=].
+                    </li>
+                    <li>[=Reject=] <var>promise</var> with a
+                    "{{NotAllowedError}}" {{DOMException}}.
+                    </li>
+                  </ol>
+                </li>
+                <li>Return <var>promise</var>.
+                </li>
+              </ol>
+              <aside class="note">
+                <p>
+                  This ensures a user can leave a document that constantly
+                  attempts to lock the pointer. The document will be blocked
+                  from locking upon initial navigation or re-acquiring lock
+                  unless the user re-engages directly with the document.
+                </p>
+                <p>
+                  Conversely, if pointer lock is exited via
+                  {{Document/exitPointerLock()}} no [=engagement gesture=] is
+                  required to reenter pointer lock. This enables applications
+                  that frequently move between interaction modes, and ones that
+                  may do so based on a timer or remote network activity.
+                </p>
+                <p>
+                  Pointer lock is commonly combined with fullscreen
```

Comment:
We should consider a provision that full screen may be required. It seems kinda bad that you can use this API without first being in full screen.

Reply:
Full screen is explicitly not a requirement, as several use cases are common in non-fullscreened applications, e.g. 9.2, 9.4.  There are e.g. medical imaging applications that use it and should not be forced to be full screen.